packages/dd-trace/src/plugins/log_plugin.js

```diff
@@ -43,12 +43,21 @@ module.exports = class LogPlugin extends Plugin {
       this.tracer.inject(span, LOG, holder)
       arg.message = messageProxy(arg.message, holder)
     })
+    this.structured = false // default unstructured logger
```

Comment:
This could be a private property if I am not mistaken :)

That way it would not potentially be exposed to the user.